packages/material-ui/src/Slider/Slider.js

```diff
@@ -120,10 +124,6 @@ const axisProps = {
     offset: percent => ({ bottom: `${percent}%` }),
     leap: percent => ({ height: `${percent}%` }),
   },
-  'vertical-reverse': {
```

Comment:
Why isn't this needed in the core?

Reply:
It might be the only advantage of high test coverage. You get a chance to spot the dead code. This logic is unreachable. There is no RTL specific handling for the vertical orientation.